server/src/rest/server.rs

```diff
@@ -34,7 +40,29 @@ pub async fn server(config: crate::config::Config) {
         )
         .route("/cargo/scan", routing::put(api::scan::scan_parcel))
         .route("/cargo/landings", routing::get(api::query::query_landings))
-        .layer(Extension(grpc_clients)); // Extension layer must be last
+        .layer(
+            CorsLayer::new()
+                .allow_origin(Any)
+                .allow_headers(Any)
+                .allow_methods(Any),
```

Comment:
I think @owlot mentioned we should have a env file indicating the CORS policies to implement.

Reply:
Yeah, just having an env var configurable allow_origin would have been nice. But I'll merge it for now. Will touch svc-cargo this week myself as well, so will add it then.